mmf/configs/zoo/datasets.yaml

```diff
@@ -264,6 +264,14 @@ vqa2:
       - url: mmf://datasets/vqa2/defaults/extras.tar.gz
         file_name: extras.tar.gz
         hashcode: e45c3dbef61e5cc550b08ed7299c88cf7937efa219d071342b8ed7a6a5642a55
+  grid:
```

Comment:
A comment here would be helpful. Do you want to really use `grid`? Seems too generic.

Reply:
Sure I will add a comment.